jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -576,26 +576,13 @@ ecma_get_internal_property (ecma_object_t *object_p, /**< object descriptor */
 ecma_property_t *
 ecma_create_named_data_property (ecma_object_t *object_p, /**< object */
                                  ecma_string_t *name_p, /**< property name */
-                                 bool is_writable, /**< 'Writable' attribute */
-                                 bool is_enumerable, /**< 'Enumerable' attribute */
-                                 bool is_configurable) /**< 'Configurable' attribute */
+                                 uint8_t prop_attributes) /**< property attributes */
```

Comment:
I like this idea, but I'd give a name to this. What do you think about `typedef uint8_t prop_attr_t;` or `typedef uint8_t prop_attr_flag_t;`?